client/components/LearnerViewDirective.js

```diff
@@ -72,15 +72,17 @@ tie.directive('learnerView', [function() {
                       <span ng-if="paragraph.isCodeParagraph()">
                         <code-snippet content="paragraph.getContent()">test</code-snippet>
                       </span>
+                      <span ng-if="paragraph.isErrorParagraph()">
+                        <a href class="tie-feedback-syntax-error-link"
+                          ng-click="showErrorString = !showErrorString">
```

Comment:
I lean against doing anything more than calling a single function in ng-click. Can we keep the toggleSyntaxErrorHint() and isSyntaxErrorShown (or perhaps syntaxErrorIsShown) terminology that we used to have? The toggleSyntaxErrorHint() function could just flip the boolean flag.